make/common/templates/notary/server-config.postgres.json

```diff
@@ -0,0 +1,28 @@
+{
+	"server": {
+		"http_addr": ":4443"
+	},
+	"trust_service": {
+		"type": "remote",
+		"hostname": "notarysigner",
+		"port": "7899",
+		"tls_ca_file": "./notary-signer-ca.crt",
+		"key_algorithm": "ecdsa"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "postgres",
+		"db_url": "postgres://server:password@postgresql:5432/notaryserver?sslmode=disable"
```

Comment:
Just want to make sure this "db_url" value change to real value in deployment

Reply:
we do not provide any configuration for notary, this will be used for internal only.  What do you mean by real value?

Reply:
I think he means in the case when DB is external.